File: src/main/java/jaysonjson/papierfuchs/object/item/ItemList.java
package jaysonjson.papierfuchs.object.item;

import jaysonjson.papierfuchs.object.item.objects.ability.CraftingUpgradekitItem;
import jaysonjson.papierfuchs.object.item.objects.ability.essence.fire.FireEs01BlazeRodItem;
import jaysonjson.papierfuchs.object.item.objects.ability.essence.fire.WallBlazeRodItem;
import jaysonjson.papierfuchs.object.item.objects.crafting.*;
import jaysonjson.papierfuchs.object.item.objects.generic.ItemWithTexture;
import jaysonjson.papierfuchs.object.item.objects.generic.resource.OreItem;
import jaysonjson.papierfuchs.object.item.objects.generic.weapon.HammerItem;
import jaysonjson.papierfuchs.object.item.objects.generic.weapon.SwordItem;
import jaysonjson.papierfuchs.object.item.objects.resource.ZoryhaFighterSword;
import jaysonjson.papierfuchs.object.item.objects.resource.ZoryhaTear;
import jaysonjson.papierfuchs.object.item.objects.resource.copper.material.CopperIngotItem;
import jaysonjson.papierfuchs.object.item.objects.resource.copper.crafting.CopperPipeItem;
import jaysonjson.papierfuchs.object.item.objects.resource.copper.crafting.CopperRodItem;
import jaysonjson.papierfuchs.object.item.objects.resource.tin.material.TinIngotItem;
import jaysonjson.papierfuchs.object.item.objects.resource.vilum.material.VilumIngotItem;
import jaysonjson.papierfuchs.object.item.objects.currency.HackSilverItem;
import jaysonjson.papierfuchs.object.item.objects.currency.ZoryhaShardItem;
import jaysonjson.papierfuchs.object.item.objects.magic.MagicPowderItem;
import jaysonjson.papierfuchs.object.item.objects.magic.MagicType;
import jaysonjson.papierfuchs.object.item.objects.other.*;
import jaysonjson.papierfuchs.object.item.objects.resource.vilum.tool.VilumSwordItem;
import jaysonjson.papierfuchs.object.item.objects.vanillaOverride.IronOreItem;
import org.bukkit.ChatColor;
import org.bukkit.Material;

public class ItemList {

    /*public static GoldBarItem GOLD_BAR = new GoldBarItem("goldBarItem", Material.GOLD_INGOT, ItemUseType.CURRENCY);
    public static GoldNuggetItem GOLD_NUGGET = new GoldNuggetItem("goldNuggetItem", Material.GOLD_NUGGET, ItemUseType.CURRENCY);*/
    public static FuchsItemRegistry<HackSilverItem> HACKSILVER = new FuchsItemRegistry<>(new HackSilverItem("hacksilver", Material.IRON_NUGGET, ItemUseType.CURRENCY));
    public static FuchsItemRegistry<SkillBookItem> SKILL_BOOK = new FuchsItemRegistry<>(new SkillBookItem("skillbook", Material.WRITTEN_BOOK, ItemUseType.OTHER));
    public static FuchsItemRegistry<ZoryhaShardItem> ZORYHA_SHARD = new FuchsItemRegistry<>(new ZoryhaShardItem("zoryha_shard", Material.NETHER_STAR, ItemUseType.CURRENCY));
    public static FuchsItemRegistry<CopperIngotItem> COPPER_INGOT = new FuchsItemRegistry<>(new CopperIngotItem("copper_ingot", Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<SilverIngotItem> SILVER_INGOT = new FuchsItemRegistry<>(new SilverIngotItem("silver_ingot", Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<MaltItem> MALT = new FuchsItemRegistry<>(new MaltItem("malt", Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<HopItem> HOP = new FuchsItemRegistry<>(new HopItem("hop", Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<GlassItem> GLASS = new FuchsItemRegistry<>(new GlassItem("glass", Material.FEATHER, ItemUseType.ABILITY));
    public static FuchsItemRegistry<LiquidContainerItem> LIQUID_CONTAINER = new FuchsItemRegistry<>(new LiquidContainerItem("liquid_container", Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<GasContainerItem> GAS_CONTAINER = new FuchsItemRegistry<>(new GasContainerItem("gas_container", Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<VilumIngotItem> VILUM_INGOT = new FuchsItemRegistry<>(new VilumIngotItem("vilum_ingot", Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<IronOreItem> IRON_ORE = new FuchsItemRegistry<>(new IronOreItem("iron_ore", Material.IRON_ORE, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<ScrapItem> SCRAP = new FuchsItemRegistry<>(new ScrapItem("scrap", Material.NETHERITE_SCRAP, ItemUseType.CURRENCY));
    public static FuchsItemRegistry<ItemWithTexture> DARK_OAK_BOWL = new FuchsItemRegistry<>(new ItemWithTexture("dark_oak_bowl", Material.FEATHER, ItemUseType.DECO, "Schüssel", 21, "§8Dunkle Eiche"));
    public static FuchsItemRegistry<ItemWithTexture> OAK_BOWL = new FuchsItemRegistry<>(new ItemWithTexture("oak_bowl", Material.FEATHER, ItemUseType.DECO, "Schüssel", 23, "§8Eiche"));
    public static FuchsItemRegistry<LegendaryChestBook> LEGENDARY_CHEST_BOOK = new FuchsItemRegistry<>(new LegendaryChestBook("legendary_chest_book", Material.WRITTEN_BOOK, ItemUseType.OTHER));
    public static FuchsItemRegistry<IDCardItem> ID_CARD = new FuchsItemRegistry<>(new IDCardItem("id_card", Material.WRITTEN_BOOK, ItemUseType.OTHER));

   /* public static BackPackItem BACKPACK_9 = new BackPackItem("backPackItem9", Material.HEART_OF_THE_SEA, ItemUseType.ABILITY, 9);
    public static BackPackItem BACKPACK_18 = new BackPackItem("backPackItem18", Material.HEART_OF_THE_SEA, ItemUseType.ABILITY, 18);
    public static BackPackItem BACKPACK_27 = new BackPackItem("backPackItem27", Material.HEART_OF_THE_SEA, ItemUseType.ABILITY, 27);
    public static BackPackItem BACKPACK_36 = new BackPackItem("backPackItem36", Material.HEART_OF_THE_SEA, ItemUseType.ABILITY, 36);
    public static BackPackItem BACKPACK_45 = new BackPackItem("backPackItem45", Material.HEART_OF_THE_SEA, ItemUseType.ABILITY, 45);
    public static BackPackItem BACKPACK_54 = new BackPackItem("backPackItem54", Material.HEART_OF_THE_SEA, ItemUseType.ABILITY, 54);
*/
    public static FuchsItemRegistry<FireEs01BlazeRodItem> FIRE_ESSENCE_01 = new FuchsItemRegistry<>(new FireEs01BlazeRodItem("fireEs01BlazeRodItem", Material.BLAZE_POWDER, ItemUseType.ABILITY));
    public static FuchsItemRegistry<WallBlazeRodItem> WALL_BLAZE_ROD = new FuchsItemRegistry<>(new WallBlazeRodItem("wallBlazeRodItem", Material.BLAZE_POWDER, ItemUseType.ABILITY));

    public static FuchsItemRegistry<AlcoholTestItem> ALCOHOL_TEST = new FuchsItemRegistry<>(new AlcoholTestItem("alcohol_tester", Material.FEATHER, ItemUseType.ABILITY));

    public static FuchsItemRegistry<MagicPowderItem> FIRE_POWDER = new FuchsItemRegistry<>(new MagicPowderItem("fire_magic_powder", Material.GLOWSTONE_DUST, ItemUseType.CRAFTING, MagicType.FIRE, 1));
    public static FuchsItemRegistry<MagicPowderItem> WATER_POWDER = new FuchsItemRegistry<>(new MagicPowderItem("water_magic_powder", Material.GLOWSTONE_DUST, ItemUseType.CRAFTING, MagicType.WATER, 4));
    public static FuchsItemRegistry<MagicPowderItem> EARTH_POWDER = new FuchsItemRegistry<>(new MagicPowderItem("earth_magic_powder", Material.GLOWSTONE_DUST, ItemUseType.CRAFTING, MagicType.EARTH, 3));
    public static FuchsItemRegistry<MagicPowderItem> AIR_POWDER = new FuchsItemRegistry<>(new MagicPowderItem("air_magic_powder", Material.GLOWSTONE_DUST, ItemUseType.CRAFTING, MagicType.AIR, 2));
    public static FuchsItemRegistry<MagicPowderItem> LIGHT_POWDER = new FuchsItemRegistry<>(new MagicPowderItem("light_magic_powder", Material.GLOWSTONE_DUST, ItemUseType.CRAFTING, MagicType.LIGHT, 5));

    public static FuchsItemRegistry<CraftingUpgradekitItem> CRAFTING_UPGRADE_KIT = new FuchsItemRegistry<>(new CraftingUpgradekitItem("crafting_upgrade_kit", Material.FEATHER, ItemUseType.ABILITY));

    public static FuchsItemRegistry<CopperPipeItem> COPPER_PIPE = new FuchsItemRegistry<>(new CopperPipeItem("copper_pipe", Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<CopperRodItem> COPPER_ROD = new FuchsItemRegistry<>(new CopperRodItem("copper_rod", Material.FEATHER, ItemUseType.CRAFTING));
    //public static CopperSwordItem COPPER_SWORD = new CopperSwordItem("copper_sword", Material.FEATHER, ItemUseType.TOOL);

    public static FuchsItemRegistry<BackPackItemNBT> BACKPACK_9_NBT = new FuchsItemRegistry<>(new BackPackItemNBT("backpack_9", Material.FEATHER, ItemUseType.ABILITY, 9));
    public static FuchsItemRegistry<BackPackItemNBT> BACKPACK_18_NBT = new FuchsItemRegistry<>(new BackPackItemNBT("backpack_18", Material.FEATHER, ItemUseType.ABILITY, 18));
    public static FuchsItemRegistry<BackPackItemNBT> BACKPACK_27_NBT = new FuchsItemRegistry<>(new BackPackItemNBT("backpack_27", Material.FEATHER, ItemUseType.ABILITY, 27));
    public static FuchsItemRegistry<BackPackItemNBT> BACKPACK_36_NBT = new FuchsItemRegistry<>(new BackPackItemNBT("backpack_36", Material.FEATHER, ItemUseType.ABILITY, 36));
    public static FuchsItemRegistry<BackPackItemNBT> BACKPACK_45_NBT = new FuchsItemRegistry<>(new BackPackItemNBT("backpack_45", Material.FEATHER, ItemUseType.ABILITY, 45));
    public static FuchsItemRegistry<BackPackItemNBT> BACKPACK_54_NBT = new FuchsItemRegistry<>(new BackPackItemNBT("backpack_54", Material.FEATHER, ItemUseType.ABILITY, 54));

    public static FuchsItemRegistry<TestSwordItem> TEST_SWORD = new FuchsItemRegistry<>(new TestSwordItem("test_sword", Material.FEATHER, ItemUseType.WEAPON));

    public static FuchsItemRegistry<BatBookItem> BAT_BOOK = new FuchsItemRegistry<>(new BatBookItem("bat_book", Material.BOOK, ItemUseType.ABILITY));

    public static FuchsItemRegistry<TinIngotItem> TIN_INGOT = new FuchsItemRegistry<>(new TinIngotItem("tin_ingot", Material.FEATHER, ItemUseType.CRAFTING));

    public static FuchsItemRegistry<HammerItem> SILVER_HAMMER = new FuchsItemRegistry<>(new HammerItem("silver_hammer", "Silber", 7, 5, -0.27f, 27, Material.FEATHER, ItemUseType.WEAPON));

    public static FuchsItemRegistry<SwordItem> TIN_SWORD = new FuchsItemRegistry<>(new SwordItem("tin_sword", ChatColor.GRAY + "Zinn", 5, 221, 1.2f, 28, Material.FEATHER, ItemUseType.WEAPON));
    public static FuchsItemRegistry<SwordItem> VILUM_SWORD = new FuchsItemRegistry<>(new VilumSwordItem("vilum_sword", ChatColor.LIGHT_PURPLE + "Vilum", 17, 120, 1.2f, 25, Material.FEATHER, ItemUseType.WEAPON));
    public static FuchsItemRegistry<SwordItem> SILVER_SWORD = new FuchsItemRegistry<>(new SwordItem("silver_sword", ChatColor.DARK_GRAY + "Silver", 8, 1225, 1.2f, 25, Material.FEATHER, ItemUseType.WEAPON));

    public static FuchsItemRegistry<OreItem> TIN_ORE = new FuchsItemRegistry<>(new OreItem("tin_ore", ChatColor.GRAY + "Zinn", -1, Material.FEATHER, ItemUseType.CRAFTING));
    public static FuchsItemRegistry<ZoryhaFighterSword> ZORYHA_FIGHTER_SWORD = new FuchsItemRegistry<>(new ZoryhaFighterSword("zoryha_fighter_sword", Material.FEATHER, ItemUseType.WEAPON));
    public static FuchsItemRegistry<ZoryhaTear> ZORYHA_TEAR = new FuchsItemRegistry<>(new ZoryhaTear("zoryha_tear", Material.FEATHER, ItemUseType.CRAFTING));

    public static FuchsItemRegistry<ItemWithTexture> UK_FLAG = new FuchsItemRegistry<>(new ItemWithTexture("uk_flag", Material.FEATHER, ItemUseType.OTHER, "English", 32, true));
    public static FuchsItemRegistry<ItemWithTexture> GERMAN_FLAG = new FuchsItemRegistry<>(new ItemWithTexture("german_flag", Material.FEATHER, ItemUseType.OTHER, "Deutsch", 31, true));

    public static FuchsItemRegistry<TestPlaceItem> TEST_PLACE = new FuchsItemRegistry<>(new TestPlaceItem("test_place", Material.DIAMOND, ItemUseType.OTHER));
    public static FuchsItemRegistry<EffectBookItem> EFFECT_BOOK = new FuchsItemRegistry<>(new EffectBookItem("effect_book", Material.FEATHER, ItemUseType.OTHER));

    public static FuchsItemRegistry<WhereWasIItem> WHERE_WAS_I = new FuchsItemRegistry<>(new WhereWasIItem("where_was_i", Material.FEATHER, ItemUseType.ABILITY));
}

File: src/main/java/jaysonjson/papierfuchs/fuchs/object/intern/item/lists/ItemList.kt
package jaysonjson.papierfuchs.fuchs.`object`.intern.item.lists

import jaysonjson.papierfuchs.fuchs.`object`.FuchsLanguageString
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.TestDualItem
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.admin.BlockChangeStickItem
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.admin.MakeVehicleStick
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.crafting.GlassItem
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.generic.*
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.other.*
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.useable.AlcoholTestItem
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.useable.BackPackItemNBT
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.useable.ChairStickItem
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.useable.CraftingUpgradekitItem
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.vanillaOverride.DefaultVanillaOverride
import jaysonjson.papierfuchs.fuchs.`object`.intern.item.objects.weapons.bows.DefaultBowItem
import jaysonjson.papierfuchs.fuchs.`object`.intern.usetype.UseTypeList
import jaysonjson.papierfuchs.fuchs.registry.FRO
import jaysonjson.papierfuchs.fuchs.registry.FuchsRegistryObject
import org.bukkit.Material


object ItemList {
   /*
    @JvmField var zoryhaShard: FRO<ZoryhaShardItem> = FRO(ZoryhaShardItem("zoryha_shard", Material.NETHER_STAR, UseTypeList.CURRENCY.copy()))
    @JvmField var malt: FRO<MaltItem> = FRO(MaltItem("malt", Material.FEATHER, UseTypeList.CRAFTING.copy()))
    @JvmField var hop: FRO<HopItem> = FRO(HopItem("hop", Material.FEATHER, UseTypeList.CRAFTING.copy()))
    @JvmField var hacksilver: FRO<HackSilverItem> = FRO(HackSilverItem("hacksilver", Material.IRON_NUGGET, UseTypeList.CURRENCY.copy()))
    @JvmField var batBook: FRO<BatBookItem> = FRO(BatBookItem("bat_book", Material.BOOK, UseTypeList.USE_ABLE.copy()))
    @JvmField var zoryhaFighterSword: FRO<ZoryhaFighterSword> = FRO(ZoryhaFighterSword("zoryha_fighter_sword", Material.FEATHER, UseTypeList.WEAPON.copy()))
    @JvmField var zoryhaTear: FRO<ZoryhaTear> = FRO(ZoryhaTear("zoryha_tear", Material.FEATHER, UseTypeList.CRAFTING.copy()))
    @JvmField var whereWasI: FRO<WhereWasIItem> = FRO(WhereWasIItem("where_was_i", Material.FEATHER, UseTypeList.USE_ABLE.copy()))
    @JvmField var scytheOfTheCorrupt: FRO<ScytheOfTheCorrupt> = FRO(ScytheOfTheCorrupt("scythe_of_the_corrupt", Material.FEATHER, UseTypeList.WEAPON.copy()))
    @JvmField var scimitarOfTheLongBlow: FRO<ScimitarOfTheLongBlow> = FRO(ScimitarOfTheLongBlow("scimitar_of_the_long_blow", Material.FEATHER, UseTypeList.WEAPON.copy()))
    @JvmField var STUPID_SHIELD: FRO<StupidShield> = FRO(StupidShield("stupid_shield", Material.FEATHER, UseTypeList.WEAPON.copy()))
    @JvmField var entitySpawnEgg: FRO<EntitySpawnEggItem> = FRO(EntitySpawnEggItem("entity_spawn_egg", Material.FEATHER, UseTypeList.USE_ABLE.copy()))
    @JvmField var molotovCocktail: FRO<MolotovCocktailItem> = FRO(MolotovCocktailItem("molotov_cocktail", Material.FEATHER, UseTypeList.PROJECTILE.copy()))
    @JvmField var phoenixSword: FRO<PhoenixSword> = FRO(PhoenixSword("phoenix_sword", Material.FEATHER))
    @JvmField var strawberry: FRO<StrawBerryItem> = FRO(StrawBerryItem("strawberry", Material.POTATO, 1, 2, FuchsLanguageString("Erdbeere")))
    @JvmField var heartCrystal: FRO<HeartCrystalItem> = FRO(HeartCrystalItem("heart_crystal", Material.FEATHER, UseTypeList.USE_ABLE.copy()))
    @JvmField var stillAlivesSword: FRO<StillAlivesSword> = FRO(StillAlivesSword("still_alive_sword", Material.FEATHER))
    @JvmField var grapplingHook: FRO<GrapplingHookItem> = FRO(GrapplingHookItem("grappling_hook", Material.FEATHER))
    @JvmField var snipersBow: FRO<SnipersBowItem> = FRO(SnipersBowItem("snipers_bow"))
    @JvmField var WHO_READS_THIS_IS_STUPID: FRO<WhoReadsThisIsStupidItem> = FRO(WhoReadsThisIsStupidItem("wrtis", Material.FEATHER))
    @JvmField var heavensScythe: FRO<HeavensScytheItem> = FRO(HeavensScytheItem("heavens_scythe", Material.FEATHER, UseTypeList.TOOL.copy()))
    @JvmField var junglesAxe: FRO<JunglesAxeItem> = FRO(JunglesAxeItem("jungles_axe", Material.IRON_AXE, UseTypeList.TOOL.copy()))
    @JvmField var communistManifesto: FRO<FuchsBookItem> = FRO(FuchsBookItem("communist_manifesto", Material.WRITTEN_BOOK, FuchsLanguageString.c(ChatColor.RED.toString() + "Communist Manifesto"), FuchsBookStrings.COMMUNIST_MANIFESTO, "Marx", ChatColor.RED.toString() + "Communist Manifesto"))
    @JvmField var lightBow: FRO<LightBowItem> = FRO(LightBowItem("light_bow"))
    @JvmField var lightArrow: FRO<LightArrowItem> = FRO(LightArrowItem("light_arrow", Material.FEATHER))
    @JvmField var grapplingYeeter: FRO<GrapplingYeeterItem> = FRO(GrapplingYeeterItem("grappling_yeeter"))
        @JvmField var pizza: FRO<FuchsFoodItem> = FRO(FuchsFoodItem("pizza", Material.POTATO, 12, 3, FuchsLanguageString("Pizza")))
    @JvmField var rice: FRO<FuchsFoodItem> = FRO(FuchsFoodItem("rice", Material.POTATO, 2, FuchsLanguageString("Reis")))
    @JvmField var doner: FRO<FuchsFoodItem> = FRO(FuchsFoodItem("doner", Material.POTATO, 2, FuchsLanguageString("Döner")))
        @JvmField var craftingUpgradeKit: FRO<CraftingUpgradekitItem> = FRO(CraftingUpgradekitItem("crafting_upgrade_kit", Material.FEATHER, UseTypeList.USE_ABLE.copy()))
    */

    @JvmField var skillBook: FRO<SkillBookItem> = FRO(SkillBookItem("skillbook", Material.WRITTEN_BOOK, UseTypeList.OTHER.copy()))
    @JvmField var liquidContainer: FRO<LiquidContainerItem> = FRO(LiquidContainerItem("liquid_container", Material.FEATHER, UseTypeList.CRAFTING.copy()))
    @JvmField var gasContainer: FRO<GasContainerItem> = FRO(GasContainerItem("gas_container", Material.FEATHER, UseTypeList.CRAFTING.copy()))

    @JvmField var legendaryChestBook: FRO<LegendaryChestBook> = FRO(LegendaryChestBook("legendary_chest_book", Material.WRITTEN_BOOK, UseTypeList.OTHER.copy()))
    @JvmField var alcoholTester: FRO<AlcoholTestItem> = FRO(AlcoholTestItem("alcohol_tester", Material.FEATHER, UseTypeList.USE_ABLE.copy()))
    @JvmField var backpack9: FRO<BackPackItemNBT> = FRO(BackPackItemNBT("backpack_9", Material.FEATHER, UseTypeList.USE_ABLE.copy(), 9, 57))
    @JvmField var backpack18: FRO<BackPackItemNBT> = FRO(BackPackItemNBT("backpack_18", Material.FEATHER, UseTypeList.USE_ABLE.copy(), 18, 57))
    @JvmField var backpack27: FRO<BackPackItemNBT> = FRO(BackPackItemNBT("backpack_27", Material.FEATHER, UseTypeList.USE_ABLE.copy(), 27, 57))
    @JvmField var backpack36: FRO<BackPackItemNBT> = FRO(BackPackItemNBT("backpack_36", Material.FEATHER, UseTypeList.USE_ABLE.copy(), 36, 57))
    @JvmField var backpack45: FRO<BackPackItemNBT> = FRO(BackPackItemNBT("backpack_45", Material.FEATHER, UseTypeList.USE_ABLE.copy(), 45, 57))
    @JvmField var backpack54: FRO<BackPackItemNBT> = FRO(BackPackItemNBT("backpack_54", Material.FEATHER, UseTypeList.USE_ABLE.copy(), 54, 57))

    @JvmField var ukFlag: FRO<FuchsFlagItem> = FRO(FuchsFlagItem("uk_flag", Material.FEATHER, 32, FuchsLanguageString("English")))
    @JvmField var germanFlag: FRO<FuchsFlagItem> = FRO(FuchsFlagItem("german_flag", Material.FEATHER, 31, FuchsLanguageString("Deutsch")))
    @JvmField var dutchFlag: FRO<FuchsFlagItem> = FRO(FuchsFlagItem("dutch_flag", Material.FEATHER, -1, FuchsLanguageString("Nederlands")))
    @JvmField var russianFlag: FRO<FuchsFlagItem> = FRO(FuchsFlagItem("russian_flag", Material.FEATHER, -1, FuchsLanguageString("русский")))
    @JvmField var italianFlag: FRO<FuchsFlagItem> = FRO(FuchsFlagItem("italian_flag", Material.FEATHER, -1, FuchsLanguageString("italiano")))

    @JvmField var effectBook: FRO<EffectBookItem> = FRO(EffectBookItem("effect_book", Material.FEATHER, UseTypeList.USE_ABLE.copy()))
    @JvmField var moneyBag: FRO<MoneyBag> = FRO(MoneyBag("money_bag", Material.FEATHER))
    @JvmField var glass: FRO<GlassItem> = FRO(GlassItem("glass", Material.FEATHER, UseTypeList.USE_ABLE.copy()))
    @JvmField var entitySpawnEgg: FRO<EntitySpawnEggItem> = FRO(EntitySpawnEggItem("entity_spawn_egg", Material.FEATHER, UseTypeList.USE_ABLE.copy()))

    @JvmField var defaultBow: FRO<DefaultBowItem> = FRO(DefaultBowItem())
    @JvmField var bountyHead: FRO<BountyHeadItem> = FRO(BountyHeadItem("bounty_head"))
    @JvmField var arrowRight: FRO<ArrowRightItem> = FRO(ArrowRightItem("arrow_right"))
    @JvmField var slotRight: FRO<SlotRightItem> = FRO(SlotRightItem())

    @JvmField var blockChangeStick: FRO<BlockChangeStickItem> = FRO(BlockChangeStickItem())
    @JvmField var makeVehicleStick: FRO<MakeVehicleStick> = FRO(MakeVehicleStick())
    @JvmField var chairStick: FRO<ChairStickItem> = FRO(ChairStickItem())

    @JvmField var soundDisc: FRO<FuchsSoundDiscItem> = FRO(FuchsSoundDiscItem())


    @JvmField var testDual: FuchsRegistryObject<TestDualItem> = FRO(TestDualItem("test_dual", Material.IRON_SWORD))

}